yt_dlp/extractor/common.py

```diff
@@ -247,6 +247,11 @@ class InfoExtractor:
                                  (For internal use only)
                                  * http_chunk_size Chunk size for HTTP downloads
                                  * ffmpeg_args     Extra arguments for ffmpeg downloader
+                    MPD formats can have the additional fields:
+                    * is_dash_periods  True if the format is the result of merging
+                                 multiple DASH periods.
+                    * dash_period_id  The period ID from the DASH manifest.
+                    * dash_period_idx  The number of the period in the DASH manifest.
```

Comment:
Unused?

Reply:
I see what happened. I was saying we could add field **when extractor needs it**. There is no point in adding this now